dataflowAPI/src/SymbolicExpansion.C

```diff
@@ -112,3 +113,17 @@ bool SymbolicExpansion::expandAarch64(SgAsmInstruction *rose_insn, BaseSemantics
     return false;
 }
 
+bool SymbolicExpansion::expandAmdgpu(SgAsmInstruction *rose_insn, BaseSemantics::RiscOperatorsPtr ops, const std::string &insn_dump) {
+    SgAsmAmdgpuInstruction *insn = static_cast<SgAsmAmdgpuInstruction *>(rose_insn);
+
+    BaseSemantics::DispatcherPtr cpu = DispatcherAMDGPU::instance(ops, 64);
+
+    try {
+        cpu->processInstruction(insn);
+    } catch (rose::BinaryAnalysis::InstructionSemantics2::BaseSemantics::Exception &e) {
```

Comment:
Do we really want to just swallow this exception?

Reply:
This is also what have been done for the rest of the architectures.(PPC64, AArch64,X86_64) and so on, where the fprintfs are commented out.
I guess the proper way would be to use some sort of debug printf instead of directly writing to stderr ?

Reply:
For this one, Dyninst should just swallow this exception. rose has more restrict requirements regarding instruction opcode, operand types, orders, and so on, and rose may also throw exception for unimplemented instructions. For all such error cases, this code will return an null AST ptr and a false return value. So, the user of Dyninst can detect this error and do corresponding error handling.

Reply:
@mxz297 Sounds good. I just wanted to make sure this wasn't an accident.